README.md

```diff
@@ -31,28 +31,77 @@ using Aqua
 Aqua.test_all(YourPackage)
 ```
 
-## Notes on Aqua 0.6
+## How to add Aqua.jl...
 
-Aqua 0.6 includes the type piracy detection, thanks to [the PR](https://github.com/JuliaTesting/Aqua.jl/pull/88) by Jakob
-Nybo Nissen (@jakobnissen) and [the original implementation](https://discourse.julialang.org/t/pirate-hunter/20402) by
-Frames Catherine White (@oxinabox).
+### ...as a test dependency?
 
-If this part of Aqua 0.6 causes a trouble, then you can disable the piracy detection
-with a flag as in `Aqua.test_all(YourPackage; piracy = false)`.
+There are two ways to add Aqua.jl as a test dependency to your package.
+To avoid breaking tests when a new Aqua.jl version is released, it is
+recommended to add a version bound for Aqua.jl.
 
-## Specifying Aqua version
+ 1. In `test/Project.toml`, add Aqua.jl to `[dep]` and `[compat]` sections, like
+    ```toml
+    [deps]
+    Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
+    Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
-To avoid breaking test when a new Aqua.jl version is released, it is
-recommended to add version bound for Aqua.jl in `test/Project.toml`:
+    [compat]
+    Aqua = "0.6"
+    ```
 
-```toml
-[deps]
-Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+ 2. In `Project.toml`, add Aqua.jl to `[compat]` and `[extras]` section and the `test` target, like
```

Comment:
I feel `Project.toml` is a little ambiguous, so it seems better with the prefix `YourPackage/`.
```suggestion
 2. In `YourPackage/Project.toml`, add Aqua.jl to `[compat]` and `[extras]` section and the `test` target, like
```